blaze/expr/tests/test_arithmetic.py

```diff
@@ -72,3 +72,15 @@ def test_dir():
 def test_arith_ops_promote_dtype():
     r = w + z
     assert r.dshape == dshape('5 * 3 * float64')
+
+
+def test_str_arith():
+    assert isinstance(cs + cs, Add)
+    assert isinstance(cs * 1, Mult)
+    assert isinstance(cs % cs, Mod)
```

Comment:
I could do `Cat`, `Format`, and `Repeat` to represent these new actions, how does that sound?